### PR TITLE
Update to build.sh to add PHY modules

### PR DESCRIPTION
**Pull request by thebsdbox**
As mentioned [here](https://www.kernel.org/doc/html/latest/networking/phy.html) the `phy` kernel subsystem is usually needed to connect drivers to underlying physical equipment. We've seen that network cards such as the Realtek 6819 (with the drivers in the initramfs) can't start when OSIE boots due to the underlying physical driver `realtek.ko` missing.

## Description

Adds physical network drivers

## Why is this needed

Some physical network adapters will cause the kernel to fail on booting.

Fixes: #

## Checklist:

I have:

- [ ] updated the documentation and/or roadmap (if required)
- [ ] added unit or e2e tests
- [ ] provided instructions on how to upgrade

### CODE REVIEW

**Review by markjacksonfishing — DISMISSED** [review]
/lgtm

**Comment by gianarb** [comment]
Thank you for your hel @thebsdbox !!!!

**Review by gianarb — DISMISSED** [review]

File: installer/alpine/build.sh [hunk 1]
```diff
@@ -14,6 +14,7 @@ build_initramfs() {
 
 	cat >/etc/mkinitfs/features.d/network.modules <<-EOF
 		kernel/drivers/net/ethernet
+		kernel/drivers/net/phy
```
Comment (mmlb) [inline installer/alpine/build.sh:17]: hmm `/phy` drivers should already be pulled as the alpine scripts are able to detect module dependencies. Maybe we don't have the correct realtek driver?

```
[~/go/src/github.com/tinkerbell/osie/installer/alpine/assets-x86_64]─[manny@dellnix]> 
mkdir chroot
                                                                                                                                                            
[~/go/src/github.com/tinkerbell/osie/installer/alpine/assets-x86_64]─[manny@dellnix]> 
tar -xf initramfs -C chroot/
no candidates

[~/go/src/github.com/tinkerbell/osie/installer/alpine/assets-x86_64]─[manny@dellnix]> 
ls chroot/lib/modules/5.4.52-1-lts/kernel/drivers/net/phy/
fixed_phy.ko  libphy.ko  phylink.ko
                                                                                                                                                            
[~/go/src/github.com/tinkerbell/osie/installer/alpine/assets-x86_64]─[manny@dellnix]> 
```

Comment (mmlb) [reply installer/alpine/build.sh:17]: I'd rather make sure we have the correct realtek driver or debug why the realtek's phy isn't being added.

Comment (thebsdbox) [reply installer/alpine/build.sh:17]: There are a number of examples of underlying `phy` drivers missing.. `lib/modules/5.4.52-1-lts/kernel/drivers/net/phy/broadcom.ko` being another.. we include all of the other `net/ethernet` drivers but not the underlying physical driver.

Comment (mmlb) [reply installer/alpine/build.sh:17]: ah very true! thanks for the reminder. lgtm, all we need is your DCO @thebsdbox

**Comment by mmlb** [comment]
Lets get the new initramfs added to this PR too. @thebsdbox are you setup to do that or want some help?

**Comment by thebsdbox** [comment]
Hi @mmlb what is needed? re:`initramfs`

**Comment by mmlb** [comment]
the opposite of `installer/alpine/skip-building-alpine-files` and `make package` I think. That should generate a new initramfs in assets-x86_64 that should be setup to be `git lfs`'d

**Comment by mmlb** [comment]
test env is failing since mholt/caddy became private and we haven't transitioned to caddyserver/caddy yet

**Comment by mmlb** [comment]
you're going to want to rebase on top of master after https://github.com/tinkerbell/osie/pull/175 lands.

**Comment by mmlb** [comment]
Actually #176.

**Comment by thebsdbox** [comment]
@mmlb this has been updated with `lfs`

**Comment by mmlb** [comment]
@thebsdbox I'm going to close this since we prefer https://github.com/tinkerbell/hook over osie's alpine. Feel free to re-open if you want to resume.

**Pull request closed without merging**